android/build.gradle

```diff
@@ -29,11 +29,10 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('TwilioVoiceReactNative_compileSdkVersion', 31)
-    buildToolsVersion safeExtGet('TwilioVoiceReactNative_buildToolsVersion', '30.0.2')
+    compileSdk safeExtGet('TwilioVoiceReactNative_compileSdkVersion', 33)
```

Comment:
Do we need to notify customers about us changing supported API versions?

Reply:
It shouldn't affect them, maybe place it in the changelog?